packages/suite/src/support/messages.ts

```diff
@@ -3158,11 +3158,40 @@ export default defineMessages({
         id: 'TR_TOR_TITLE',
         defaultMessage: 'Tor',
     },
+    TR_TOR_ENABLE: {
+        id: 'TR_TOR_ENABLE',
+        defaultMessage: 'Enable Tor',
+    },
+    TR_TOR_DISABLE: {
+        id: 'TR_TOR_DISABLE',
+        defaultMessage: 'Disable Tor',
+    },
+    TR_TOR_ENABLE_AND_CONFIRM: {
+        id: 'TR_TOR_ENABLE_AND_CONFIRM',
+        defaultMessage: 'Enable Tor and confirm',
```

Comment:
This has nothing to do with this text. Just a product question.

Does it make sense to add more places where users can enable/disable tor? We have anchor links now. In settings, you can enable/disable tor but also you can change Tor address, port, and open trezor.io links as .onion links. Moreover, there is `Learn more` link.